src/features/contactmoment/service.ts

```diff
@@ -32,7 +37,25 @@ export function useContactmomentService(): {
       }
     });
   };
+
+  const getGespreksResultaten = () => {
```

Comment:
Voor de consistentie is het denk ik mooi als je hier een ServiceData object retourneert. Dan hoef je niet zelf refs bij te houden in de vue component voor error en loading. Nu is er alleen nog ServiceResult.fromFetcher, die de swr library gebruikt voor caching en pollen, maar als je dat niet nodig hebt kan je ook een simpelere functie maken, bv ServiceResult.fromPromise. Als we dan op een later moment toch caching/polling willen toevoegen, hoef je de vue component niet aan te passen.

Reply:
Zoiets:
```ts
fromPromise<T = unknown>(promise: Promise<T>) {
    const result = ServiceResult.loading<T>();

    promise
      .then((r) => {
        Object.assign(result, {
          state: "success",
          data: r,
        });
      })
      .catch((e) => {
        Object.assign(result, {
          state: "error",
          error: e instanceof Error ? e : new Error(e),
        });
      });

    return result;
  }
```

Reply:
bedoel je een tweede functie toevoegen aan ServiceResult naast fromFetcher? of alleen SericeResult gebruiken voor de success/loading/error properties?